packages/valaxy-theme-yun/node/index.ts

```diff
@@ -30,11 +28,19 @@ export function themePlugin(options: ResolvedValaxyOptions): Plugin {
         optimizeDeps: {
           exclude: ['@docsearch/js'],
         },
-
         valaxy: {},
       }
     },
   }
 }
 
-export default themePlugin
+// TODO: Whether to provide defineConfig function definition
+function themeYun(options: ResolvedValaxyOptions): ValaxyThemeOptions {
```

Comment:
I prefer to the form.

```ts
export default defineThemePlugin((options) => {
  return {
    vite: {},
  }
})
```

Reply:
Yes, I think so too, defineThemePlugin can reduce a lot of processes